frontend/src/components/Gallery.tsx

```diff
@@ -30,6 +33,15 @@ export default function Gallery(){
             <br/>
                 <input type= "range" min= "1" max = "250" defaultValue={rangeValue}
                        onChange={(e) => setRangeValue(e.target.value)}/>
+             <br/>
+            <br/>
```

Comment:
I would suggest using css instead html layout tags like <br/>
for example you could give each input a margin: 
```css
.Gallery > input[type~="range"] {
 margin: 1rem;
}
.Gallery > input[type~="radio"] {
 margin: 1rem;
}
```

Of course this would be easier if you gave every input a class.